### PR TITLE
add back character actions statistic

### LINKED ISSUE #1564: add back character ability usage to viewer page
min, max, average

### PR DESCRIPTION
**Pull request by k0l11**
closes #1564 

- actions are always sorted according to an (arbitrary) pre-defined order that is similar to docs
- consistency with docs is not ensured and color choice may not be perfect
- legend order + mapping between color and action is consistent across sims

Example images: 
grafik
grafik

### CODE REVIEW

File: ui/packages/ui/src/Pages/Viewer/Components/Miscellaneous/CharacterActionsBarChart/BarChart.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+type Props = {
+  width: number;
+  height: number;
+  names?: string[];
+  actions?: SourceStats[];
+  actionNames?: string[] | null;
+}
+
+const margin = { top: 0, left: 300, right: 20, bottom: 40 };
```
Comment (unleashurgeek) [inline ui/packages/ui/src/Pages/Viewer/Components/Miscellaneous/CharacterActionsBarChart/BarChart.tsx:15]: Should set the left margin similar to the Character DPS bar graphs (I need to write something that is more dynmaic and based on max number of characters in label or something).

**Comment by unleashurgeek** [comment]
this graph also probably doesn't need full width and can split with the field time pie 50/50?

**Comment by k0l11** [comment]
new images:
grafik
grafik

**Review by unleashurgeek — APPROVED** [review]
nit: for added clarity, might be nice for this graph to also have an x-axis label of something like "count"

**Pull request merged**